aggregator/aggregator.go

```diff
@@ -69,6 +69,7 @@ type RpcAggregatorer interface {
 }
 
 type RestAggregatorer interface {
+	GetStateRoot(rollupId uint32, blockHeight uint64) (*types.GetStateRootResponse, error)
```

Comment:
Whats the difference between this new function and the `GetStateRootUpdateAggregation`? Why do we need a new endpoint?

Reply:
Same as above.

Reply:
https://github.com/Nuffle-Labs/nffl/pull/302/commits/39317dd6adf8aa7b2a8e5d576f00451065537ea7 & https://github.com/Nuffle-Labs/nffl/pull/302/commits/119506078ac821d02fac3db4637615b2a827162f